pages/press.md

```diff
@@ -698,7 +698,7 @@ Are you looking to get a speaker or interview about Solid? Contact us on [press@
 
 2020-02-24 [Berners-Lee's Solid project: Schneier joins team to give you back control over data](https://www.zdnet.com/article/berners-lees-solid-project-schneier-joins-team-to-give-you-back-control-over-data/)
 
-2020-02-24 [10 things in tech you need to know today](https://www.businessinsider.my/10-things-in-tech-you-need-to-know-today-february-24-2020-2)
+<!------- 2020-02-24 [10 things in tech you need to know today](https://www.businessinsider.my/10-things-in-tech-you-need-to-know-today-february-24-2020-2) ---------->
```

Comment:
The build gets this error:

>  Liquid Exception: invalid byte sequence in UTF-8 in pages/press.md

I'm not sure what it could be, but I wonder if that's due to the number of dashes here. I don't think so, but to be sure:

```suggestion
<!-- 2020-02-24 [10 things in tech you need to know today](https://www.businessinsider.my/10-things-in-tech-you-need-to-know-today-february-24-2020-2) -->
```